WordPress/src/main/res/layout/fragment_me.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingLeft="@dimen/margin_medium"
+    android:paddingRight="@dimen/margin_medium">
+
+    <org.wordpress.android.widgets.WPNetworkImageView
+        android:id="@+id/me_avatar"
+        android:layout_width="@dimen/avatar_sz_large"
+        android:layout_height="@dimen/avatar_sz_large"
+        android:layout_alignParentTop="true"
+        android:layout_centerHorizontal="true"
+        android:layout_marginTop="@dimen/me_avatar_margin_top"
+        android:elevation="@dimen/card_elevation" />
```

Comment:
From [this screenshot](https://cloudup.com/ifWeFLNm2H4) I think it's supposed to be circular. It'd awesome if you can take care of that and ping me in the PR (or send me the commit if you are not going to send a PR), so I can check out how you did it.